src/serlo-editor/plugins/fill-in-the-gap-exercise/index.tsx

```diff
@@ -0,0 +1,87 @@
+import { FillInTheGapRenderer } from './renderer'
+import { PluginToolbar } from '@/serlo-editor/editor-ui/plugin-toolbar'
+import { TextEditorFormattingOption } from '@/serlo-editor/editor-ui/plugin-toolbar/text-controls/types'
+import {
+  type EditorPlugin,
+  type EditorPluginProps,
+  object,
+  child,
+  string,
+} from '@/serlo-editor/plugin'
+import { selectDocument, useAppSelector } from '@/serlo-editor/store'
+import { EditorPluginType } from '@/serlo-editor-integration/types/editor-plugin-type'
+
+export const fillInTheGapExercise: EditorPlugin<FillInTheGapExerciseState> = {
+  Component: FillInTheGapExerciseEditor,
+  config: {},
+  state: createFillInTheGapExerciseState(),
+}
+
+export type FillInTheGapExerciseState = ReturnType<
+  typeof createFillInTheGapExerciseState
+>
+
+function createFillInTheGapExerciseState() {
+  return object({
+    text: child({
+      plugin: EditorPluginType.Text,
+      config: {
+        formattingOptions: [
+          TextEditorFormattingOption.code,
+          TextEditorFormattingOption.colors,
+          TextEditorFormattingOption.headings,
+          TextEditorFormattingOption.katex,
+          TextEditorFormattingOption.links,
+          TextEditorFormattingOption.lists,
+          TextEditorFormattingOption.math,
+          TextEditorFormattingOption.paragraphs,
+          TextEditorFormattingOption.richTextBold,
+          TextEditorFormattingOption.richTextItalic,
+          TextEditorFormattingOption.gap,
```

Comment:
could we extend this from the `defaultFormattingOptions`?

Reply:
Good idea!